packages/auth/src/graphql/resolvers/grant.test.ts

```diff
@@ -481,38 +482,46 @@ describe('Grant Resolvers', (): void => {
     })
 
     test('Returns error for unknown grant', async (): Promise<void> => {
-      const gqlQuery = appContainer.apolloClient
-        .mutate({
-          mutation: gql`
-            query GetGrant($id: String!) {
-              grant(id: $id) {
-                id
-                client
-                state
-                access {
+      try {
```

Comment:
```suggestion
      expect.assertions(3)
      try {
```
to make sure we don't end up ignoring the expect's in the try